books/free-programming-books-subjects.md

```diff
@@ -299,10 +299,12 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 ### Graphics Programming
 
 * [3D Game Shaders For Beginners](https://github.com/lettier/3d-game-shaders-for-beginners) - David Lettier (Git) [(HTML)](https://lettier.github.io/3d-game-shaders-for-beginners)
+* [Augmented Reality](https://www.researchgate.net/publication/227164365_Augmented_Reality_An_Overview) - Julie Carmigniani and Borko Furht
 * [Blender 3D: Noob to Pro](https://en.wikibooks.org/wiki/Blender_3D%3A_Noob_to_Pro) - Wikibooks
 * [Blender Manual](https://docs.blender.org/manual/en/latest)
 * [Computer Graphics from scratch](http://gabrielgambetta.com/computer-graphics-from-scratch) - Gabriel Gambetta (:construction: *in process*)
 * [DirectX manual](http://user.xmission.com/~legalize/book/download/index.html) (draft)
+* [Extended Reality](https://www.researchgate.net/publication/362525560_A_Review_of_Extended_Reality_XR_Technologies_in_the_Future_of_Human_Education_Current_Trend_and_Future_Opportunity) - Ahmed Jamah Ahmed Alnagrat
```

Comment:
Don't invent titles! Use the title given by the resource.
This is a journal article, not a book. We don't list journal articles.